SwerveControllerCommand/robotcontainer.py

```diff
@@ -0,0 +1,116 @@
+#
+# Copyright (c) FIRST and other WPILib contributors.
+# Open Source Software; you can modify and/or share it under the terms of
+# the WPILib BSD license file in the root directory of this project.
+#
+
+from wpilib import XboxController
+import commands2
+import constants
+from subsystems.drivesubsystems import DriveSubsystem
+from wpimath.trajectory import Trajectory, TrajectoryConfig, TrajectoryGenerator
+from wpimath.geometry import Translation2d, Pose2d, Rotation2d
+from wpimath.trajectory.constraint import DifferentialDriveVoltageConstraint
+from wpimath.controller import ProfiledPIDController, PIDController, HolonomicDriveController
+import math
+
+
+class RobotContainer:
+    def __init__(self):
+        """
+        The container for the robot. Contains subsystems, OI devices, and commands.
+        """
+        self.robotDrive = DriveSubsystem()
+
+        # Driver Controller
+        self.driverController = XboxController(
+            constants.OIConstants.kDriverControllerPort
+        )
+
+        """
+        Configure default commands
+        """
```

Comment:
Same here regarding the comment difference.
```suggestion
        # Configure default commands
```